cmd/syncthing/gui.go

```diff
@@ -276,7 +276,7 @@ func (s *apiService) Serve() {
 	getRestMux.HandleFunc("/rest/db/status", s.getDBStatus)                      // folder
 	getRestMux.HandleFunc("/rest/db/browse", s.getDBBrowse)                      // folder [prefix] [dirsonly] [levels]
 	getRestMux.HandleFunc("/rest/folder/versions", s.getFolderVersions)          // folder
-	getRestMux.HandleFunc("/rest/folder/pullerrors", s.getPullErrors)            // folder
+	getRestMux.HandleFunc("/rest/folder/fileerrors", s.getFileErrors)            // folder
```

Comment:
As we're anyway cleaning up and clarifying, why not just `/rest/folder/errors`? The intention is to cover all sorts of errors that happen in any phase of the sync operation right?

Reply:
Yes, it covers errors at any point during scanning or syncing associated with a file (item). Errors "on the folder as a whole" (e.g. watcher error, home disk full) are not reported on this endpoint. That was my intention of leaving "file" in there, but trying to represent this tiny distinction in the name is possibly more confusing than helpful.